docs/vtex-io/Hello/welcome.md

```diff
@@ -8,6 +8,6 @@ excerpt: "VTEX IO is a low-code development platform that simplifies the deliver
 ---
 [block:html]
```

Comment:
Não consigo deixar sugestão para esse bloco, mas tem os seguintes typos:
- concpts
- "tolearn"

Reply:
Eu recomendaria algo assim "Explore our VTEX IO documentation for guides and concepts. For practical learning, try our hands-on Learning Center."